packages/website/plugins/generated-rule-docs.ts

```diff
@@ -34,6 +34,59 @@ function nodeIsParent(node: unist.Node<unist.Data>): node is unist.Parent {
   return 'children' in node;
 }
 
+const prettierConfig = resolveConfig.sync(__dirname);
```

Comment:
[Nitpick] Now we're mixing our style & the default, heh.

![Screenshot of array-type mixing single and double apostrophes](https://user-images.githubusercontent.com/3335181/232327578-af2e7145-4946-4d4e-901c-b216a8ae240d.png)